thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -345,6 +406,34 @@ export default Controller.extend({
       });
   }),
 
+  /**
+   * Concurrency task to reload page once a running replay is complete
+   * @param {Number} jobId - the id for the newly triggered replay job
+   * @return {undefined}
+   */
+  checkForNewAnomalies: task(function * (jobId) {
+    yield timeout(5000);
+
+    // In replay status check, continue to display "pending" banner unless we have known success or failure.
+    fetch(`/detection-onboard/get-status?jobId=${jobId}`).then(checkStatus)
+      .then((jobStatus) => {
+        const replayStatusObj = _.has(jobStatus, 'taskStatuses')
+          ? jobStatus.taskStatuses.find(status => status.taskName === 'FunctionReplay')
+          : null;
+        if (replayStatusObj) {
+          if (replayStatusObj.taskStatus.toLowerCase() === 'completed') {
+            this.transitionToRoute({ queryParams: { repRunStatus: null }});
+          } else {
+            this.get('checkForNewAnomalies').perform(jobId);
+          }
+        }
+      })
+      .catch(() => {
+        // If we have job status failure, go ahead and transition to full alert page.
```

Comment:
Are we surfacing this error to the user?

Reply:
Yes. Not the error details, but they will see an error page.